Assets/Scripts/GameManager.cs

```diff
@@ -25,13 +27,21 @@ void Update()
             currentSanity = currentSanity - 1.0f;
         }
     }
-    public void lastScene() {
+
+    public void loadScene(int nextSceneNum)
+    {
+        UnityEngine.SceneManagement.SceneManager.LoadScene(nextSceneNum);
+    }
+    public void lastScene()
+    {
         sceneNum -= 1;
     }
-    public void nextScene() {
+    public void nextScene()
+    {
         sceneNum += 1;
     }
-    public void reset() {
+    public void reset()
+    {
         sceneNum = 0;
     }
```

Comment:
Are you still using these? If you're using loadScene check that you don't need those other ones, remove them otherwise to keep code clean.

Reply:
yes I will remove that ASAP